frontend/src/views/Seedlot/SeedlotRegistrationForm/index.tsx

```diff
@@ -4,28 +4,115 @@ import {
   FlexGrid,
   Row,
   Breadcrumb,
-  BreadcrumbItem
+  BreadcrumbItem,
+  Button
 } from '@carbon/react';
+import { ArrowRight, CheckmarkOutline } from '@carbon/icons-react';
 
 import PageTitle from '../../../components/PageTitle';
 import SeedlotRegistrationProgress from '../../../components/SeedlotRegistrationProgress';
 import OrchardStep from '../../../components/SeedlotRegistrationSteps/OrchardStep';
-import InterimStorage from '../../../components/SeedlotRegistrationProgress/InterimStorage';
+import InterimStorage from '../../../components/SeedlotRegistrationSteps/InterimStep';
 import OwnershipStep from '../../../components/SeedlotRegistrationSteps/OwnershipStep';
-
+import { AllStepData } from './definitions';
+import {
+  initInterimState,
+  initOrchardState,
+  initOwnershipState
+} from './utils';
+import { SingleOwnerForm } from '../../../components/SeedlotRegistrationSteps/OwnershipStep/definitions';
 import './styles.scss';
+import InterimForm from '../../../components/SeedlotRegistrationSteps/InterimStep/definitions';
+import { SeedlotOrchard } from '../../../types/SeedlotTypes/SeedlotOrchard';
```

Comment:
Move these imports before the style import.

Reply:
[done!](https://github.com/bcgov/nr-spar-webapp/pull/82/commits/e861fab5cf1c64c81e0240b14d672a182dca02c0)